### DISCUSSION TITLE
Add a simple standard way of toggling between normal level and maximum level?

### DISCUSSION THREAD

**Discussion by phil-s** [magit/transient#243] [Ideas]
The 'level' functionality via `C-x l` is handy for setting a default, but the UI feels cumbersome whenever I want to check whether a feature exists on some currently-hidden level.  As far as I know, to use an option which is not at my current level I need to use `C-x l` and then either change the overall level, or change the level of the specific option, and then quit back to the regular menu, and finally do the thing.  And if I don't want that option as a default, I then need to go through the same process to undo it afterwards.

What I'd like is a standard key binding to temporarily enable the maximum level so that I will immediately be able to see and use *any* available option.  That would let me keep the default level set to cover my day-to-day options, and still give me easy access to everything else.

Alternatively, much like `M-1` .. `M-4` are bound to the section visibility commands in Magit, maybe `M-1` .. `M-7` could be bound to "change to level N" in transient menus?  Or even simply `1` .. `7`?

**Comment by tarsius** [comment]
I've implemented this (the boolean toggle) in 92572454b5f022a0c05f1de590987168bbf63242. It's available on `C-x a`.
